### ISSUE TITLE
Streamline bulk tab managing, possible multi-row select bug on clean profiles

### ISSUE THREAD

**Issue by rayman89** [jaszhix/tab-master-5000-extension#6]
Could you make it possible to re-arrange, close tabs, etc in bulk? Like for example enabling you to have a check mark on the tab tiles so when you select many and then press the "x" on one of them it close all the checked tabs or if you drag and drop one of the checked tabs it moves with it all the tabs that were checked as well.

**Comment by jaszhix** [comment]
In the table view, as mentioned in the FAQ in About -> Support, you can hold the CTRL key, or use SHIFT to select multiple tabs, then right click actions will apply to all of the selections. This hasn't been implemented for the tile view yet, however.

**Comment by rayman89** [comment]
Hmm would be nice to be able to do the same on both views. I dont think people are goign to read the support section unless tryng to trobleshoot something. In my opinion the features have to be intuitive/easy to find. 

BTW I tested on a new clean profile with just you addon installed and noticed that the selected tabs do not show the fact that they are selected and the selection is not smooth sometimes it fails to select the tabs.

**Comment by jaszhix** [comment]
Agreed, there is a number of UX issues I've been wanting to address, as the extension was intended to be unapologetically for power users initially, but since v2 I've been attempting to make it easier to use.

That may be an issue with the default "Chromesque" theme, it seems to work fine on a fresh profile after the theme is switched.
